StaticThreadController.h

```diff
@@ -0,0 +1,65 @@
+/*
+	StaticThreadController.h - Controlls a list of Threads with different timings
+
+	Basicaly, what it does is to keep track of current Threads and run when
+	necessary.
+
+	StaticThreadController is an extended class of Thread, because of that,
+	it allows you to add a StaticThreadController inside another kind of ThreadController...
+
+	It works exact as ThreadController except you can't add or remove treads dynamically.
+
+	Created by Alex Eremin, September, 2016.
+	Released into the public domain.
+*/
+
+#ifndef StaticThreadController_h
+#define StaticThreadController_h
+
+#include "Thread.h"
+
+template <int N>
+class StaticThreadController: public Thread{
+protected:
+	Thread thread[N];
```

Comment:
My bad, it must hold pointers, otherwise this simple implementation is useless and dangerous.

Initially I wanted to make static container which holds everything without any additional overhead, but just realized that it has to be some kind of std::tuple + lots of template metaprogramming to be able to safely hold any derived types.